### PR TITLE
First tranche of plugin pages, aligning our plugins pages with our do…

### PR DESCRIPTION
**Pull request by samnixon87**
Aligning our plugins pages with our docs pages

Preview URL: https://deploy-preview-1388--roadie.netlify.app/backstage/plugins/

### CODE REVIEW

**Comment by Xantier** [comment]
"Tranche", learn new stuff everyday

File: content/backstage/plugins/incident-io.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+humanName: Incident.io
+heading: 'Backstage Incident.io Plugin'
+lead: 'Display incidents associated to Backstage components'
+attribution:
+  text: Incident.io
+  href: https://incident.io/
+
+intro: |
+  <p>
+    As a prerequisite, you need to have configured the Incident.io catalog-importer to sync with your Backstage catalog. You can visit your incident.io dashboard to create a custom field that is powered by the Backstage Component catalog type.
```
Comment (Xantier) [inline content/backstage/plugins/incident-io.md:11]: Can we link to the "catalog-importer", whatever that is?

Comment (Xantier) [reply content/backstage/plugins/incident-io.md:11]: I see it's linked below, maybe worth wrapping the relevant words in this text into <a> also

File: content/backstage/plugins/launchdarkly.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+      yarn add @roadiehq/backstage-plugin-launchdarkly
+
+  - intro: Add proxy configurations in the app-config.yaml file
+    language: 'yaml'
+    code: |
+      // app-config.yaml
+      proxy:
+        # ...
+        '/launchdarkly/api':
+        target: https://app.launchdarkly.com/api
```
Comment (Xantier) [inline content/backstage/plugins/launchdarkly.md:33]: here extra indent as well needed

**Comment by dtuite** [comment]
Looks great. Def ready to merge once the indentation issues Jussi spotted are fixed.

File: content/backstage/plugins/launchdarkly.md (hunk 2 above)
Comment (samnixon87) [reply content/backstage/plugins/launchdarkly.md]: Nice catch - done.

File: content/backstage/plugins/incident-io.md (hunk 1 above)
Comment (samnixon87) [reply content/backstage/plugins/incident-io.md]: Makes sense. Done

File: content/backstage/plugins/coder.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+    code: yarn --cwd packages/app add @coder/backstage-plugin-coder
+
+  - intro: Add proxy config to your app-config.yaml
+    language: typescript
+    code: |
+      proxy:
+        endpoints:
+          '/coder':
+          # Replace with your Coder deployment access URL (add a trailing slash)
+          target: 'https://coder.example.com/'
```
[the remark this answers is not in the document]
Comment (samnixon87) [reply content/backstage/plugins/coder.md]: Nice catch - done.

**Review by Xantier — APPROVED** [review]

**Pull request merged**